src/envs/painting.py

```diff
@@ -501,7 +531,15 @@ def _get_tasks(self, num_tasks: int, num_objs_lst: List[int],
                 state.set(self._robot, "fingers", 0.0)
                 state.set(target_obj, "grasp", grasp)
                 state.set(target_obj, "held", 1.0)
-                assert self._OnTable_holds(state, [target_obj])
+                if isinstance(self, envs.RepeatedNextToPaintingEnv):
```

Comment:
why does this need to be specific to RepeatedNextToPaintingEnv? could we just remove the if and keep the change below?